tests/helpers/dataset.py

```diff
@@ -20,15 +20,24 @@
 import time
 import re
 
-def get_tmp_ds_name(mlq_size=7, llq_size=7):
+
+def get_tmp_ds_name(mlq_size=7, llq_size=7, symbols=False):
     """ Function or test to ensure random names of datasets
     the values of middle and last qualifier can change size by parameter,
-    but by default includes one letter."""
+    but by default includes one letter.
+    Also includes indication if symbols should be in the string, default=false."""
     ds = "ANSIBLE" + "."
-    ds += "P" + get_random_q(mlq_size).upper() + "."
+    if symbols:
+        ds += "P" + get_random_qs(mlq_size).upper() + "."
+    else:
+        ds += "P" + get_random_q(mlq_size).upper() + "."
     ds += "T" + str(int(time.time()*1000))[-7:] + "."
-    ds += "C" + get_random_q(llq_size).upper()
-    return ds
+    if symbols:
+        ds += "C" + get_random_qs(llq_size).upper()
+    else:
+        ds += "C" + get_random_qs(llq_size).upper()
```

Comment:
I think this function should be get_random_q only right? because this is without symbols

Reply:
oops!  yep, I copied it wrong